index.js

```diff
@@ -148,8 +149,22 @@ async function handleAlert(location, alertType) {
         fallbackMessage: `An alert to check on the bathroom at ${location.displayName} was not responded to. Please check on it`,
         fallbackToPhoneNumbers: location.fallbackNumbers,
         fallbackFromPhoneNumber: location.twilioNumber,
+        sirenParticleId: location.siren_particle_id,
```

Comment:
The `alertInfo` object is only used as a parameter into `braveAlerter.startAlertSession`. So it doesn't need to have `sirenParticleId` in here and it can be moved into the same `else if` block as `braveAlerter.startAlertSession(alertInfo)`.